kernel/src/hil/i2c.rs

```diff
@@ -27,19 +28,27 @@ pub enum Error {
 
     /// No error occured and the command completed successfully.
     CommandComplete,
+
+    /// Error reported when a virtual device fails synchronously
+    Request(ErrorCode),
```

Comment:
I am not sure what "request" means here, could this be something like `VirtualizationErr`?

Reply:
That would be an error returned if the actual request to the I2C would not be successful. Even though it is only in the case of virtualization, it is not an actual virtualization error. At first, I named it ErrorCode (`ErrorCode(ErrorCode)`). Maybe CommandError ?

Reply:
This has been settled.